coinoffers.py

```diff
@@ -112,13 +112,15 @@ def localbitcoins():
 
 
 def coinmotion():
-    response = requests.get('http://coinmotion.com/rates?ajax_content=true')
-    offer = response.json()
-    return [{'exchange': 'coinmotion',
-             'price': Decimal('1.02') * Decimal(offer['btc']['sell']),
-             'max_amount': Decimal('2.0'),
-             'link': 'https://coinmotion.com/buy',
-             'seller': 'coinmotion.com'}]
+    try:
+        offer = requests.get('http://coinmotion.com/rates?ajax_content=true').json()
+        return [{'exchange': 'coinmotion',
+                 'price': Decimal('1.02') * Decimal(offer['btc']['sell']),
+                 'max_amount': Decimal('2.0'),
+                 'link': 'https://coinmotion.com/buy',
+                 'seller': 'coinmotion.com'}]
+    except ValueError:
+        print("Decoding json have failed")
```

Comment:
Printing to stdout will corrupt the actual JSON output from the script. In case of an exception, the script should probably either crash with no JSON output, or log the failure on stderr (or, even better, use the `logging` framework).

If we opt for logging errors and producing partial JSON output, it would make sense to include information about failed requests in the JSON output. The list of offers would have to be wrapped separately from status output in the JSON structure.

Also, it would be useful to have the invalid Coinmotion response stored on disk to ease debugging the failed JSON decoding.

What do you think about these ideas?